examples/webgl_layers.html

```diff
@@ -0,0 +1,156 @@
+		<script src='js/libs/dat.gui.min.js'></script>
+
+		<script>
+
+			var container, stats;
+			var camera, scene, renderer;
+
+			var radius = 100, theta = 0;
+
+			init();
+			animate();
+
+			function init() {
+
+				container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				var info = document.createElement( 'div' );
+				info.style.position = 'absolute';
+				info.style.top = '10px';
+				info.style.width = '100%';
+				info.style.textAlign = 'center';
+				info.innerHTML = '<a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> webgl - layers';
+				container.appendChild( info );
+
+				var bgTexture = new THREE.TextureLoader().load( 'textures/brick_diffuse.jpg' );
+				var bgColor = new THREE.Color( 0xf0f0f0 );
+
+				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 1, 10000 );
+				camera.layers.enable( 0 );
+				camera.layers.enable( 1 );
+				camera.layers.enable( 2 );
+
+				scene = new THREE.Scene();
+				scene.background = bgColor;
+
+				var light = new THREE.DirectionalLight( 0xffffff, 1 );
+				light.position.set( 1, 1, 1 ).normalize();
+				light.layers.enable( 0 );
+				light.layers.enable( 1 );
+				light.layers.enable( 2 );
+
+				scene.add( light );
+
+				var colors = [ 0xff0000, 0x00ff00, 0x0000ff ];
+				var geometry = new THREE.BoxBufferGeometry( 20, 20, 20 );
+				var layer;
+
+				for ( var i = 0; i < 300; i ++ ) {
+
+					layer = ( i % 3 );
+
+					var object = new THREE.Mesh( geometry, new THREE.MeshLambertMaterial( { color: colors[ layer ] } ) );
+
+					object.position.x = Math.random() * 800 - 400;
+					object.position.y = Math.random() * 800 - 400;
+					object.position.z = Math.random() * 800 - 400;
+
+					object.rotation.x = Math.random() * 2 * Math.PI;
+					object.rotation.y = Math.random() * 2 * Math.PI;
+					object.rotation.z = Math.random() * 2 * Math.PI;
+
+					object.scale.x = Math.random() + 0.5;
+					object.scale.y = Math.random() + 0.5;
+					object.scale.z = Math.random() + 0.5;
+
+					object.layers.set( layer );
+
+					scene.add( object );
+
+				}
+
+				raycaster = new THREE.Raycaster();
+
+				renderer = new THREE.WebGLRenderer();
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				container.appendChild(renderer.domElement);
+
+				stats = new Stats();
+				container.appendChild( stats.dom );
+
+				var layers = { red: true, green: true, blue: true, background: false };
+
+				//
+				// Init gui
+				var gui = new dat.GUI();
+				gui.add( layers, 'red' ).onChange( function () { camera.layers.toggle( 0 ); } );
+				gui.add( layers, 'green' ).onChange( function () { camera.layers.toggle( 1 ); } );
+				gui.add( layers, 'blue' ).onChange( function () { camera.layers.toggle( 2 ); } );
+
+				gui.add( layers, 'background' ).onChange( function ( value ) { scene.background = value ? bgTexture : bgColor; } );
```

Comment:
I would remove the background since it feels out of place in this example. Let's just focus on layers.